test/tests.luau

```diff
@@ -1699,6 +1699,25 @@ TEST("world:delete() invokes OnRemove hook", function()
 		
 		world:delete(B)
 
+		CHECK(called)
+	end
+	do CASE "#3"
+		local world = world_new()
+		local pair = jecs.pair
+
+		local viewingContainer = world:component()
```

Comment:
In hindsight I think world:entity for viewingContainer should work. Try that codepath I am curious anyways.